client/components/upstreamMessages/html/src/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Upstream Messages Sample Integrations - PayPal Web SDK</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+  <body>
+    <h1>Upstream Messages Integrations</h1>
+
+    <ul>
+      <li>
+        <a href="/recommended/index.html">Recommended integration</a>
+      </li>
```

Comment:
I'm thinking this landing page should include the minimal example, similar to the `/components/paypalPayments` `custom` example:
```suggestion
      <li>
        <a href="/recommended/index.html">Recommended integration</a>
      </li>
      <li>
        <a href="/minimal/index.html">Recommended integration</a>
      </li>
```